src/msg/async/Stack.h

```diff
@@ -235,8 +248,10 @@ class Worker {
   Worker(CephContext *c, unsigned worker_id)
     : cct(c), perf_logger(NULL), id(worker_id), references(0), center(c) {
```

Comment:
No need to initialize `perf_logger` here since it's always initialized in the constructor.  This is pre-existing but better changed for symmetry with `perf_labeled_logger`:
```suggestion
    : cct(c), id(worker_id), references(0), center(c) {
```